graphene_django/fields.py

```diff
@@ -103,39 +102,44 @@ def resolve_connection(cls, connection, default_manager, args, iterable):
 
     @classmethod
     def connection_resolver(
-        cls,
-        resolver,
-        connection,
-        default_manager,
-        max_limit,
-        enforce_first_or_last,
-        root,
-        info,
-        **args
-    ):
-        first = args.get("first")
-        last = args.get("last")
-
-        if enforce_first_or_last:
-            assert first or last, (
-                "You must provide a `first` or `last` value to properly paginate the `{}` connection."
-            ).format(info.field_name)
-
-        if max_limit:
-            if first:
-                assert first <= max_limit, (
-                    "Requesting {} records on the `{}` connection exceeds the `first` limit of {} records."
-                ).format(first, info.field_name, max_limit)
-                args["first"] = min(first, max_limit)
-
-            if last:
-                assert last <= max_limit, (
-                    "Requesting {} records on the `{}` connection exceeds the `last` limit of {} records."
-                ).format(last, info.field_name, max_limit)
-                args["last"] = min(last, max_limit)
-
-        iterable = resolver(root, info, **args)
-        on_resolve = partial(cls.resolve_connection, connection, default_manager, args)
+            cls,
+            resolver,
+            connection,
+            default_manager,
+            max_limit,
+            enforce_first_or_last,
+            root,
+            info,
+            **kwargs):
+        # pylint: disable=R0913,W0221
+
+        first = kwargs.get("first")
+        last = kwargs.get("last")
+        if not (first is None or first > 0):
```

Comment:
What do you think of wording this as `if first is not None and first < 1:` or even `if first and first < 1:`? I think it might be clearer, but I'm not sure.

Reply:
Changed to `if first is not None and first <= 0`.
0 is also a invalid value so `if first and first < 1` is not correct.